core/src/main/java/nl/nn/adapterframework/stream/Message.java

```diff
@@ -49,7 +48,7 @@
 import nl.nn.adapterframework.util.StreamUtil;
 import nl.nn.adapterframework.util.XmlUtils;
 
-public class Message implements Serializable {
+public class Message implements Serializable, AutoCloseable {
```

Comment:
Waarom heb je AutoCloseable toegevoegd?
Ik denk dat ik dat niet wil, omdat ik niet steeds 'gewaarschuwd' wil worden dat Message in een try with resource constructie zou moeten zitten.

Reply:
Als de Ladybug een checkpoint die een stream is gaat stubben dan wordt er niks met de originele stream (een Message object in geval van FF) gedaan (die wordt niet uitgelezen) omdat die vervangen wordt door een stream gebaseerd op de message in het originele report. Zie ook:

https://github.com/ibissource/ibis-ladybug/blob/master/src/main/java/nl/nn/testtool/Report.java#L433

Reply:
ok